llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -1405,10 +1403,10 @@ class VPIRInstruction : public VPRecipeBase {
     return true;
   }
 
-  /// Update the recipes first operand to the last lane of the operand using \p
-  /// Builder. Must only be used for VPIRInstructions with at least one operand
-  /// wrapping a PHINode.
-  void extractLastLaneOfFirstOperand(VPBuilder &Builder);
+  /// Update the recipe's first operand to the final lane of the operand using
+  /// \p Builder. Must only be used for VPIRInstructions with at least one
+  /// operand wrapping a PHINode.
+  void extractFinalLaneOfFirstOperand(VPBuilder &Builder);
```

Comment:
Why the name change? Perhaps I'm missing something, but I don't know really what the difference is between `Last` and `Final`. If this is related to ExtractLast, etc. then surely `Last` is more in-keeping with the operation?